ansible/roles/test/vars/testcases.yml

```diff
@@ -215,52 +215,52 @@ testcases:
     port_toggle:
       filename: port_toggle.yml
       vtestbed_compatible: no
-      topologies: [t0, t0-64, t0-64-32, t0-116, t1, t1-lag, t1-64-lag, t1-64-lag-clet, ptf32, ptf64]
+      topologies: [t0, t0-64, t0-64-32, t0-116, t1, t1-lag, t1-64-lag, ptf32, ptf64]
+
+    qos_sai_mlnx:
+      filename: qos_sai_mlnx.yml
+      topologies: [ptf32, ptf64, t1, t1-lag, t0, t0-64, t0-116]
 
-    qos:
```

Comment:
Same as above, this change is not related to this PR. Looks like it was made by mistake.